fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs

```diff
@@ -8123,19 +8148,49 @@ mod tests {
             }
 
             fn get_flaps_left_position_percent(&mut self) -> f64 {
-                self.read_by_name("LEFT_FLAPS_POSITION_PERCENT")
+                let values: Vec<f64> = vec![
```

Comment:
I know this is in the tests but I can't hold back myself 😄
Don't need to construct a vector for this (same for the other functions below):
```suggestion
                let values: &[f64] = &[
```